test-cases/gutenberg/videopress.md

```diff
@@ -0,0 +1,114 @@
+
+# VideoPress Block - Test Cases
```

Comment:
I noticed that the Story block resides in `test-cases/jetpack`. Would it make sense to relocate VideoPress to that location or relocate Story to `test-cases/gutenberg`? The current structure is a bit confusing.

Reply:
Ah, good catch. Definitely, the VideoPress block test cases should be under `jetpack` folder not `gutenberg`. I'll open a PR to correct this. Thanks for the call out 🙇 !

Reply:
[Here is the PR with the fix.](https://github.com/wordpress-mobile/test-cases/pull/140)